.cirrus.yml

```diff
@@ -242,7 +242,7 @@ alpine_task:
 # We aim to support both the current and previous macOS release.
 macos_monterey_task:
   macos_instance:
-    image: monterey-xcode-13.1
+    image: ghcr.io/cirruslabs/macos-monterey-base:latest
```

Comment:
Interesting! Details here: https://cirrus-ci.org/guide/macOS/#available-images

Reply:
Definitely. I've been meaning to look at that Tart tool mentioned there since I got my M1 machine but I never get around to it.